pages/api/createRecord.ts

```diff
@@ -0,0 +1,39 @@
+import { verifyIdToken, getFirebaseAdmin } from "next-firebase-auth";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+type Response = { status: string; message: string };
+
+function writeRecord() {
+  const db = getFirebaseAdmin().database();
```

Comment:
We can connect to the db via `next-firebase-auth`'s `getFirebaseAdmin` method. Without it, we'd need to initialize the firebase client again